tests/projects/test_docker_projects.py

```diff
@@ -130,10 +131,15 @@ def test_docker_invalid_project_backend_local():
 
 
 @pytest.mark.parametrize("artifact_uri, host_artifact_uri, container_artifact_uri, should_mount", [
-    ("/tmp/mlruns/artifacts", "/tmp/mlruns/artifacts", "/tmp/mlruns/artifacts", True),
+    (os.path.join(os.path.sep, "tmp", "mlruns", "artifacts"),
```

Comment:
```suggestion
    os.path.abspath(os.path.join("tmp", "mlruns", "artifacts")),
```

To construct an absolute path here and elsewhere (vs trying to do it by prefixing the path with the path-separator)